packages/framework/src/Component/Console/ConsoleExitCodeListener.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Console;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Symfony\Component\Console\Event\ConsoleTerminateEvent;
+
+/**
+ * Taken from https://github.com/VasekPurchart/Console-Errors-Bundle/blob/master/src/Console/ConsoleExitCodeListener.php
+ * because so far, the bundle does not support PHP 8, @see https://github.com/VasekPurchart/Console-Errors-Bundle/issues/11
+ */
+class ConsoleExitCodeListener
```

Comment:
wouldn't be better to change it to a subscriber?

Reply:
fixed in c448b33f8